lib/src/pie_menu_core.dart

```diff
@@ -299,4 +303,38 @@ class _PieMenuCoreState extends State<PieMenuCore>
       _bounceController.reverse();
     });
   }
+
+  void _attachMenu(
+    bool rightClicked, {
+    PieMenuOpenEvent? menuOpenEvent,
```

Comment:
I've added the `PieMenuOpenEvent? menuOpenEvent` here for you to hook into and use the event's alignment and displacement.